src/components/LogButton.js

```diff
@@ -1,26 +1,44 @@
 import React, { Component } from 'react';
-import { Card, Paragraph } from 'react-native-paper';
+import { Button, Paragraph, Title } from 'react-native-paper';
+import { StyleSheet } from 'react-native';
 //REDUX
 import { connect } from 'react-redux';
 import { incrementCount } from '../redux/actions';
+import { hasReachedDailyLimitSelector } from '../redux/selectors';
+
+const styles = StyleSheet.create({
+    habitButton: {
+        padding: 20,
+    },
+    habitButtonText: {
+        display: 'block',
+        textAlign: 'center',
+        fontSize: '15px',
```

Comment:
Same is true for string values as font sizes - looks like they need to be numbers

Reply:
Update above